test/specs/footer.ts

```diff
@@ -0,0 +1,114 @@
+import { expect } from "@wdio/globals";
+import axios from "axios";
+import MainPage from "../pageobjects/main.page.ts";
+import * as testData from "../data/testdata.json"
+
+
+describe("Rentzila", () => {
+  it("C214-Verify that all elements on the footer are displayed and all links are clickable", async () => {
+    await MainPage.open();
+
+    await expect(MainPage.logo).toBeExisting();
+    await expect(MainPage.footerLogo).toBeExisting();
+    await expect(MainPage.footerLogo).not.toBeClickable();
+    await expect(MainPage.aboutUsLabel).toBeExisting();
+    await expect(MainPage.aboutUsLabel).toHaveText("Про нас");
+    await expect(MainPage.privacyPolicyLink).toBeExisting();
+    await expect(MainPage.privacyPolicyLink).toHaveAttribute(
+      "href",
+      "/privacy-policy/"
+    );
+    await expect(MainPage.cookiePolicyLink).toBeExisting();
+    await expect(MainPage.cookiePolicyLink).toHaveAttribute(
+      "href",
+      "/cookie-policy/"
+    );
+    await expect(MainPage.termsConditionsLink).toBeExisting();
+    await expect(MainPage.termsConditionsLink).toHaveAttribute(
+      "href",
+      "/terms-conditions/"
+    );
+    await expect(MainPage.forBuyersLabel).toBeExisting();
+    await expect(MainPage.forBuyersLabel).toHaveText("Користувачам");
+    await expect(MainPage.productsLink).toBeExisting();
+    await expect(MainPage.productsLink).toHaveAttribute("href", "/products/");
+    await expect(MainPage.tendersLink).toBeExisting();
+    await expect(MainPage.tendersLink).toHaveAttribute("href", "/tenders-map/");
+    await expect(MainPage.requestsLink).toBeExisting();
+    await expect(MainPage.requestsLink).toHaveAttribute(
+      "href",
+      "/requests-map/"
+    );
+    await expect(MainPage.contactsLabel).toBeExisting();
+    await expect(MainPage.contactsLabel).toHaveText("Контакти");
+    await expect(MainPage.emailLink).toBeExisting();
+    await expect(MainPage.emailLink).toHaveAttribute(
+      "href",
+      "mailto:info@rentzila.com.ua"
+    );
+  });
+
+  it('C226-"У Вас залишилися питання?" form', async () => {
```

Comment:
@Anastas11aZakharova This test case failed for me as well. By the way, add it to the separate file, as it was discussed in one of the comments about the specs pattern.

Reply:
As discussed with Bohdan Alieinykov, the issue is in the modal pop up window that behaves randomly in WDIO. We tried multiple approaches to handle this, even copied wdio.conf and ts.config fully from a real project. Non of this worked unfortunately .
Agreed to wait for new chrome and WDIO version to see if it becomes more stable.